readme.md
**szpakowski-portfolio
Projekt porzucony.


changelog.txt
= 1.1.6=
* commment checkbox css improved.
* parallax issue fixed in chrome.

= 1.1.5=
* style improved.

= 1.1.4=
* Fontawesome Updated.

= 1.1.3=
* Style Improved.

= 1.1.2=
* one click home page setup bug fixed.

= 1.1.1=
* one click home page setup.

= 1.1.0=
* Customizer popup option added.

= 1.0.9 =
* scrolling issue fixed.

= 1.0.8 =
* js issue fixed.
* All issue fixed.

= Version: 1.0.7 =
* Style Improve.

= Version: 1.0.6 =
* escping issue fixed.
* reame file issue fixed.

= Version: 1.0.5 =
1. Page serach authr name,date and category issue fixed.
2. header color and background color issue fixed.

= Version: 1.0.4 =
1. Customizer background feature added.
2. Comment responsive style fixed.
3. Header image and color issue fixed.

= Version: 1.0.3 =
1. Style Improved.
2. Section order changed.

= Version: 1.0.2 =
1. css style Improved.
2. error fixed.

Version: 1.0.0 =

bck.php
<?php echo do_shortcode('
			[skillwrapper type="bar"]
			[skill title_background="#f7a53b" bar_foreground="#f7a53b" bar_background="#eeeeee" percent="90" title="CSS"]
			[/skillwrapper]') ?>
			<ul class="portfolio_skills">
				<li>Pasja Informatyki(pl) - Kurs CSS - 270 min</li>
				<li>Derek Banas(en) - Kurs RW in One Video - 30 min</li>
				<li>Strona w czystym html i css - 700 podstron</li>
			</ul>

			<?php echo do_shortcode('
			[skillwrapper type="bar"]
			[skill title_background="#3B56A5" bar_foreground="#062FAA" bar_background="#eeeeee" percent="95" title="HTML"]
			[/skillwrapper]') ?>
			<ul class="portfolio_skills">
				<li>Pasja Informatyki(pl) - Kurs HTML - 370 min</li>
				<li>Strona w czystym html i css - 700 podstron</li>
			</ul>

			<?php echo do_shortcode('
			[skillwrapper type="bar"]
			[skill title_background="#fa6e6e" bar_foreground="#fa6e6e" bar_background="#eeeeee" percent="75" title="PHP"]
			[/skillwrapper]') ?>

			<ul class="portfolio_skills">
				<li>Pasja Informatyki(pl) - Kurs PHP - 420 min</li>
				<li>LearnWebCode(en) - Kurs Wordpress - 330 min</li>
				<li>Strona na Wordpressie - portfolio</li>
				<li>CodinGame (SzpaQ) - Zadania Średnie - 1</li>
			</ul>

			<?php echo do_shortcode('
			[skillwrapper type="bar"]
			[skill title_background="#fa6e6e" bar_foreground="#fa6e6e" bar_background="#eeeeee" percent="75" title="JavaScript"]
			[/skillwrapper]') ?>

			<ul class="portfolio_skills">
				<li>Pasja Informatyki(pl) - Kurs JS - 270 min</li>
			</ul>

			<?php echo do_shortcode('
			[skillwrapper type="bar"]
			[skill title_background="#f7a53b" bar_foreground="#f7a53b" bar_background="#eeeeee" percent="90" title="MySQL"]
			[/skillwrapper]') ?>
			<ul class="portfolio_skills">
				<li>Pasja Informatyki(pl) - Kurs MySQL - 180 min</li>
			</ul>

			<?php echo do_shortcode('
			[skillwrapper type="bar"]
			[skill title_background="#336699" bar_foreground="#336699" bar_background="#eeeeee" percent="40" title="C++"]
			[/skillwrapper]') ?>

			<ul class="portfolio_skills">
				<li>Pasja Informatyki(pl) - Kurs C++ - 480 min</li>
				<li>Pasja Informatyki(pl) - Kurs Podejcia Obiektowego (C++) - 380 min</li>
				<li>Polski Spoj (Szpakusik) - Zadania Łatwe - 15</li>
				<li>CodinGame (SzpaQ) - Zadania Średnie - 1</li>
			</ul>



			<?php echo do_shortcode('
			[skillwrapper type="bar"]
			[skill title_background="#336699" bar_foreground="#336699" bar_background="#eeeeee" percent="40" title="Java"]
			[/skillwrapper]') ?>
			<ul class="portfolio_skills">
				<li>Derek Banas(en) - Kurs Java in One Video - 30 min</li>
				<li>Polski Spoj (Szpakusik) - Zadania Łatwe - 1</li>
				<li>CodinGame (SzpaQ) - Zadania Łatwe - 5</li>
				<li>CodinGame (SzpaQ) - Zadania Średnie - 1</li>
			</ul>

			

			

loop.php
<!-- Start the Loop. -->
<?php
if (have_posts()) : while (have_posts()) : the_post(); ?>
<div id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
  <div class="blog-loop">
        <?php if ( 'post' == get_post_type() ) : ?>

    <div class="post-details">
      <ul class="post-meta-ul">
        <li class="author"><i class="fa fa-user"></i><?php the_author_posts_link(); ?></li>
        <li class="post-date"><a href="<?php esc_url(the_permalink()); ?>"><i class="fa fa-calendar"></i><?php echo get_the_date(); ?></li>
      </ul>
    </div>
      <?php endif; ?>

    <div class="post-title">
      
      <h1><?php the_title();?></h1>
      <div class="post-thumbnail-div"><?php the_post_thumbnail('featuredlite-custom-thumb'); ?></div>
    </div>
      <?php if ( 'post' == get_post_type() ) : ?>

    <div class="post-meta">
      <ul>
        <li class="category"><i class="fa fa-folder-open-o"></i><?php the_category(' / '); ?></li>
        <li class="comments"><a href="<?php comments_link(); ?>"><i class="fa fa-comments-o"></i><?php comments_number(__('No Comments', 'featuredlite'), __('One Comment', 'featuredlite'),  __('% Comments', 'featuredlite') ); ?></a></li>
      </ul>
    </div>
  <?php endif; ?>
    <div class="post-content">
      <a href="<?php the_permalink() ?>">  </a>
      <?php the_excerpt();?>
    </div>
    <div class="read-more"><a class="read_more" href="<?php the_permalink() ?>"><?php echo _e('Read More','featuredlite'); ?></a></div>
    
  </div>
</div>
<?php
endwhile;
featuredlite_pagination();
else:?>
<div class="post-none">
  <p>
    <?php _e('sorry no post matched your criteria!', 'featuredlite'); ?>
  </p>
</div>
<?php endif; ?>
<!--End post-->

comments.php
<?php
/**
* Comments form and comment feature
*/
?>
<?php
if (!empty($_SERVER['SCRIPT_FILENAME']) && 'comments.php' == basename($_SERVER['SCRIPT_FILENAME']))
die (__('Please do not load this page directly. Thanks!','featuredlite'));
if ( post_password_required() ) { ?>
<p class="nocomments"><?php _e('This post is password protected. Enter the password to view comments.','featuredlite'); ?></p>
<?php return; } ?>
<!-- You can start editing here. -->
<div class="clear"></div>
<div class="comment-section" id="commentsbox">
     <div class="post-comment-top-border"></div>
     <?php if ( have_comments() ) : ?>
     <h1 class="post-info" id="comments">
     <?php comments_number(__('No Responses', 'featuredlite'), __('Comments...', 'featuredlite'),  __('% Responses', 'featuredlite') ); ?>
     </h1>
     <ol class="commentlist">
          <?php wp_list_comments(); ?>
     </ol>
     <div class="comment-nav">
          <div class="alignleft">
               <?php previous_comments_link(); ?>
          </div>
          <div class="alignright">
               <?php next_comments_link(); ?>
          </div>
     </div>
     <?php else : // this is displayed if there are no comments so far ?>
     <?php if ( comments_open() ) : ?>
     <!-- If comments are open, but there are no comments. -->
     <?php else : // comments are closed ?>
     <!-- If comments are closed. -->
     <p class="nocomments"><?php _e('Comments are closed.', 'featuredlite'); ?></p>
     <?php endif; ?>
     <?php endif; ?>
     <?php if ( comments_open() ) :
     echo "<div id='comments_pagination'>";
          paginate_comments_links(array('prev_text' => __('&laquo;','featuredlite'), 'next_text' => __('&raquo;','featuredlite')));
     echo "</div>";
     $custom_comment_field = '<p class="comment-form-comment"><textarea id="comment" name="comment" cols="45" rows="8" aria-required="true"></textarea></p>';  //label removed for cleaner layout
     comment_form(array(
     'comment_field'               => $custom_comment_field,
     'comment_notes_after'         => '',
     'logged_in_as'                => '',
     'comment_notes_before'        => '',
     'title_reply'                 => __('Zostaw komentarz', 'featuredlite'),
     'cancel_reply_link'           => __('Porzuć', 'featuredlite'),
     'label_submit'                => __('Wyślij', 'featuredlite')
     ));
     endif; // if you delete this the sky will fall on your head ?>
</div>

functions.php
<?php
/**
 *  function Page
 */
if ( ! isset( $content_width ) ) {
  $content_width = 1170;
}
function featuredlite_setup() {
load_theme_textdomain('featuredlite', get_template_directory() . '/languages');

// Add RSS feed links to <head> for posts and comments.
add_theme_support( 'automatic-feed-links' );
/*
     * Switch default core markup for search form, comment form, and comments
     * to output valid HTML5.
     */
    add_theme_support( 'html5', array('comment-form', 'comment-list', 'gallery', 'caption'
) );
/*
* Let WordPress manage the document title.
* By adding theme support, we declare that this theme does not use a
* hard-coded <title> tag in the document head, and expect WordPress to
* provide it for us.
*/
add_theme_support( 'title-tag' );
add_theme_support( 'custom-logo', array(
    'height'      => 60,
    'width'       => 225,
    'flex-height' => true,
  ) ); 

/**
 * Enable support for Post Thumbnails on posts and pages.
 * @param  
 * @return mixed|string
 */
   
add_theme_support('post-thumbnails');
add_image_size( 'featuredlite-custom-thumb', 800, 450, true );

/* woocommerce support */
add_theme_support( 'woocommerce' );

// post-header image
$defaults = array(
    'default-image'          => get_template_directory_uri().'/images/bg.jpg',
    'width'                  => 0,
    'height'                 => 0,
    'flex-height'            => false,
    'flex-width'             => false,
    'uploads'                => true,
    'random-default'         => false,
    'header-text'            => false,
    'default-text-color'     => 'f16c20',
    'wp-head-callback'       => '',
    'admin-head-callback'    => '',
    'admin-preview-callback' => '',
);
add_theme_support( 'custom-header', $defaults );  
add_editor_style( 'custom-editor-style.css' );
$args = array(
  'default-color' => 'ffffff',
);
add_theme_support( 'custom-background', $args );
// Recommend plugins
        add_theme_support( 'recommend-plugins', array(
            'themehunk-customizer' => array(
                'name' => esc_html__( 'ThemeHunk Customizer', 'featuredlite' ),
                'active_filename' => 'themehunk-customizer/themehunk-customizer.php',
            ),
            'lead-form-builder' => array(
                'name' => esc_html__( 'Lead Form Builder', 'featuredlite' ),
                'active_filename' => 'lead-form-builder/lead-form-builder.php',
            ),
            'woocommerce' => array(
                'name' => esc_html__( 'Woocommerce', 'featuredlite' ),
                'active_filename' => 'woocommerce/woocommerce.php',
            )
        ) );
}
add_action( 'after_setup_theme', 'featuredlite_setup' );
require_once( get_template_directory() . '/inc/index.php' );
/**
 * Enqueue scripts and styles for the front end.
 */
function featuredlite_scripts() {
$featuredlite_animation = get_theme_mod('featuredlite_animation_disable');
// Add Genericons font, used in the main stylesheet.
if($featuredlite_animation =='' || $featuredlite_animation =='0'){
wp_enqueue_style( 'animate', get_template_directory_uri() . '/css/animate.css', array(), '1.0.0' );
}
wp_enqueue_style( 'shopline-font-awesome', get_template_directory_uri() . '/css/font-awesome/css/fontawesome-all.css', array(), '1.0.0' );
wp_enqueue_style( 'shopline-font-awesome-old', get_template_directory_uri() . '/css/font-awesome/css/font-awesome.css', array(), '1.0.0' );
wp_enqueue_style( 'owl-carousel', get_template_directory_uri() . '/css/owl.carousel.css', array(), '1.0.0' );
wp_enqueue_style('featuredlite-style', get_stylesheet_uri());
wp_add_inline_style( 'featuredlite-style', featuredlite_custom_header() );
// js include
wp_enqueue_script( 'wow', get_template_directory_uri() . '/js/wow.js', array( 'jquery' ), '', true );
wp_enqueue_script( 'easing', get_template_directory_uri() . '/js/easing.js', array( 'jquery' ), '', true );
wp_enqueue_script( 'classie', get_template_directory_uri() . '/js/classie.js', array( 'jquery' ), '', true );
wp_enqueue_script( 'owl-carousel', get_template_directory_uri() . '/js/owl.carousel.js', array( 'jquery' ), '', true );
wp_enqueue_script( 'featuredlite-custom', get_template_directory_uri() . '/js/custom.js', array( 'jquery' ), '', true );
 // Comment reply
if (is_singular() && get_option('thread_comments')){
    wp_enqueue_script('comment-reply');
  }
}
add_action( 'wp_enqueue_scripts', 'featuredlite_scripts' );

/* Porfolio page */

//adding custom.css
function featuredlite_custom_css() {
    wp_enqueue_style( 'custom', get_template_directory_uri() . '/css/custom.css', array(), '1.0.0' );
    wp_enqueue_style( 'animate', get_template_directory_uri() . '/css/animate.css', array(), '1.0.0' );
}
add_action('wp_enqueue_scripts', 'featuredlite_custom_css');



function featuredlite_edit_porfolio($wp_customize){
//portfolio img
$wp_customize -> add_section('featuredlite_portfolio', array(
        'title' => 'Portfolio'
    ));
$wp_customize -> add_setting('featuredlite_portfolio_image');
$wp_customize -> add_control(new WP_Customize_Cropped_Image_Control($wp_customize, 'featuredlite_portfolio_image_control', array(
        'label' => 'Image',
        'section' => 'featuredlite_portfolio',
        'settings' => 'featuredlite_portfolio_image',
        'width' => 500,
        'height' => 500,
    )));

    /* Portfolio H1 text */

$wp_customize -> add_setting('featuredlite_portfolio_h1');
$wp_customize -> add_control(new WP_Customize_Control($wp_customize, 'featuredlite_portfolio_h1_control', array(
        'label' => 'Headline',
        'section' => 'featuredlite_portfolio',
        'settings' => 'featuredlite_portfolio_h1',
        'default' => 'Skill Set',
)));
}
add_action('customize_register', 'featuredlite_edit_porfolio'); 
?>



page-kontakt.php
<?php
/**
* The template for displaying all pages
*
* This is the template that displays all pages by default.
* Please note that this is the WordPress construct of pages and that
* other 'pages' on your WordPress site will use a different template.
*
* @package ThemeHunk
* @subpackage Featuredlite
* @since Featuredlite 1.0
*/
get_header();
?>
<?php $layout = 'right'; ?>
<div id="page" class="<?php echo $layout; ?>">
	<div class="content-wrapper">
		<div class="content">
		<div class="breadcrumb breadcrumb-center">
			<h1><?php the_title(); ?></h1>
		</div>
			<?php if (have_posts()) : while (have_posts()) : the_post();?>
			<div class="page-description">
				<div class="contact-parent">
					<h6>Skontaktuj się używając podanych danych lub natychmiastowo skorzystaj z formularza kontaktowego!</h6>

					<p>Numer telefonu:</p>
					<a href="tel:570546193"><p class="p-right"> 570 - 546 - 193</p></a>
					<p>E-mail:</p>
					<a href="mailto:szpakowski.n0@gmail.com"><p class="p-right last"> szpakowski.n0@gmail.com</p></a>
				</div>
				<?php the_content(); ?>
				
			</div>
			<div class="multipage-links">
				<?php
					wp_link_pages( array(
								'before'      => '<span class="meta-nav">' . __( 'Pages:', 'featuredlite' ) . '</span>',
								'after'       => '',
								'link_before' => '<span class="active">',
								'link_after'  => '</span>',
					) );
				?>
			</div>
			<?php
			// If comments are open or we have at least one comment, load up the comment template.
			if ( comments_open() || get_comments_number() ) {
			comments_template();
			}
			endwhile;
			endif;
			?>
		</div>
		 <?php if ( $layout != 'no-sidebar' ) { ?>
		<div class="sidebar-wrapper"><!--Sidebar wrapper start-->
		<?php get_sidebar(); ?>
	</div>
	<?php } ?>
</div>
</div>
<?php get_footer(); ?>

page-portfolio.php
<?php
/**
* The template for displaying all pages
*
* This is the template that displays all pages by default.
* Please note that this is the WordPress construct of pages and that
* other 'pages' on your WordPress site will use a different template.
*
* @package ThemeHunk
* @subpackage Featuredlite
* @since Featuredlite 1.0
*/
get_header();
?>
<?php $layout = "no-sidebar"; ?>
<div id="page" class="<?php echo $layout; ?>">
	<div class="content-wrapper">
		<div class="content">
	<!-- Custom -->
			<div class="portfolio-image-div">
				<img class="circle-image" src="<?php echo wp_get_attachment_url(get_theme_mod('featuredlite_portfolio_image')) ?>">
			</div>
	<!-- -->
		<div class="breadcrumb breadcrumb-center portfolio">
			<h1 class="animated2 fadeIn"><?php echo get_theme_mod('featuredlite_portfolio_h1') ?></h1>
		</div>
			<?php if (have_posts()) : while (have_posts()) : the_post();?>
			<div class="page-description">
				<div class="flip-container">
					<div class="flip-card php"></div>
					<div class="flip-card html"></div>
					<div class="flip-card css"></div>
					<div class="flip-card java"></div>
					<div class="flip-card sql"></div>
					<div class="flip-card cpp"></div>
					<div class="flip-card js"></div>
					<div class="flip-card bootstrap"></div>
					<div class="flip-card json"></div>
					<!-- <div class="flip-card"></div>-->


					<div class="clearfix"></div>

				</div>

			

						<!-- END OF SKILLSET-->

				<div class="breadcrumb breadcrumb-center">
				<h1>Realizacje</h1>
				</div>
				<div class="team-fix">
					<?php get_template_part( 'template/section_team');?>
				</div>

					<div class="breadcrumb breadcrumb-center">
						<h1>Umiejętności interpersonalne</h1>
					</div>

				<?php echo do_shortcode('
				[skillwrapper type="circle" track_color="#dddddd" chart_color="#333333" chart_size="150" align="center"]
				[skill percent="45" title="Przemawianie"]
				[skill percent="85" title="J. Angielski"]
				[skill percent="70" title="Praca w Zespole"]
				[skill percent="90" title="Nawiązywanie Kontaktów"]
				[/skillwrapper]') ?>
			</div>



			<div class="multipage-links">
				<?php
					wp_link_pages( array(
								'before'      => '<span class="meta-nav">' . __( 'Pages:', 'featuredlite' ) . '</span>',
								'after'       => '',
								'link_before' => '<span class="active">',
								'link_after'  => '</span>',
					) );
				?>
			</div>
			<?php
			// If comments are open or we have at least one comment, load up the comment template.
			if ( comments_open() || get_comments_number() ) {
			comments_template();
			}
			endwhile;
			endif;
			?>
		</div>
		 <?php if ( $layout != 'no-sidebar' ) { ?>
		<div class="sidebar-wrapper"><!--Sidebar wrapper start-->
		<?php get_sidebar(); ?>
	</div>
	<?php } ?>
</div>
</div>
<?php get_footer(); ?>